static/js/interactive-map/NewMap.js

```diff
@@ -108,6 +108,7 @@ class NewMap extends ANSWERS.Component {
       .withPadding(this.config.mapPadding)
       .build();
     this.map = map;
+    window.mappp = this.map._map.map;
```

Comment:
I don't have much context but what is this.map._map.map, and could we call it something other than window.mappp? I tried to trace the code and it looks like this.map._map is a ProviderMapOptions, but I couldn't find this.map inside of the ProviderMapOptions class. Maybe putting some method onto MapOptions to get this property would make it a little clearer?

Reply:
This was totally meant for personal debugging! Removing